src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/LarsDataService.cs

```diff
@@ -173,18 +173,23 @@ private static CategorisedQualifications GetCategorisedQualifications(ICollectio
             return qualificationSet;
         }
 
-        private static ICollection<FrameworkMetaData> FilterFrameworks(IEnumerable<FrameworkMetaData> frameworks)
+        private ICollection<FrameworkMetaData> FilterFrameworks(IEnumerable<FrameworkMetaData> frameworks)
         {
             var progTypeList = new[] { 2, 3, 20, 21, 22, 23 };
 
             return frameworks.Where(s => s.FworkCode > 399)
```

Comment:
What's the significance of 399?  Is this best put into a variable name explaining what 399 is?

Reply:
Fixed, now it is a const